### PR TITLE
Create indy crypto generating ansible role

### PR DESCRIPTION
**Pull request by VladimirDzamba**

### CODE REVIEW

File: platforms/hyperledger-indy/configuration/roles/create/helm_component/tasks/main.yaml [hunk 1]
```diff
@@ -26,9 +26,41 @@
...
     name: "{{ playbook_dir }}/../../shared/configuration/roles/helm_lint"
   vars:
     helmtemplate_type: "{{ component_type }}"
     chart_path: "{{ gitops.chart_source }}"
-    value_file: "{{ release_dir }}/{{ component_name }}/{{ component_type }}.yaml"
\ No newline at end of file
+    value_file: "{{ release_dir }}/{{ component_name }}/{{ component_type }}.yaml"
+
+##############################################################################################
+# Generate Indy Crypto for trustee
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/roles/create/helm_component/tasks/main.yaml]: Missing endorser crypto

File: platforms/hyperledger-indy/configuration/roles/check/crypto/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+##############################################################################################
+# This role check trustee and stewards crypto in Vault
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/roles/check/crypto/tasks/main.yaml]: missing endorsers

**Comment by VladimirDzamba** [comment]
Getting vault token into variable in role is still in progress

File: platforms/hyperledger-indy/configuration/deploy-network.yaml [hunk 3]
```diff
@@ -36,11 +36,27 @@
...
     when: network['type'] == 'indy' and organizationItem.type == 'peer'
 
   # delete build directory
   - name: Remove build directory
     file:
       path: "./build"
       state: absent
+
+  # Generate indy crypto and insert into Vault
+  - name: 'Generate indy crypto and insert into Vault'
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/deploy-network.yaml]: Wrong order, crypto has to be generated before node startup! Please follow https://alm.accenture.com/wiki/display/BLOCKOFZ/Sequence+Diagrams

File: platforms/hyperledger-indy/configuration/roles/create/helm_component/crypto/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+##############################################################################################
+# Generate Indy Crypto for trustee
+- name: Trustee crypto generating
+  template:
+    src: "{{ dlt_templates[component_type] }}"
+    dest: "{{ values_file }}"
+  vars:
+    identity_name: "{{ trusteeItem.name }}"
+    vault_path: "{{ organization }}.trustee"
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/roles/create/helm_component/crypto/tasks/main.yaml]: trustees

File: platforms/hyperledger-indy/configuration/roles/create/helm_component/crypto/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+##############################################################################################
+# Generate Indy Crypto for endorser
+- name: Endorser crypto generating
+  template:
+    src: "{{ dlt_templates[component_type] }}"
+    dest: "{{ values_file }}"
+  vars:
+    identity_name: "{{ endorserItem.name }}"
+    vault_path: "{{ organization }}.endorser"
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/roles/create/helm_component/crypto/tasks/main.yaml]: endorsers

File: platforms/hyperledger-indy/configuration/roles/setup/node/tasks/main.yaml [hunk 6]
```diff
@@ -42,8 +42,20 @@
   loop_control:
     loop_var: stewardItem
 
+# ---------------------------------------------------------------------
```
Comment (juraj-durech) [inline platforms/hyperledger-indy/configuration/roles/setup/node/tasks/main.yaml]: why is this here?
